Sources/SwiftDocC/Infrastructure/Link Resolution/PathHierarchy.swift

```diff
@@ -119,7 +119,17 @@ struct PathHierarchy {
                     allNodes[id, default: []].append(node)
                 }
             }
-            
+
+            for relationship in graph.relationships where relationship.kind == .overloadOf {
+                // An 'overloadOf' relationship points from symbol -> group. We want to disfavor the
+                // individual overload symbols in favor of resolving links to their overload group
+                // symbol.
+                guard let sourceNode = nodes[relationship.source] else {
+                    continue
+                }
+                sourceNode.isDisfavoredInCollision = true
```

Comment:
If we only need to access the source node once we can access the optional value directly 
```suggestion
                nodes[relationship.source]?.isDisfavoredInCollision = true
```

Reply:
Huh! I didn't realize you could mutate through the optional access like that. Thanks!